src/configs/esbuild.es5.config.js

```diff
@@ -20,6 +20,7 @@
 const buildHelpers = require('../helpers/build')
 const alias = require('../plugins/esbuild-alias')
 const babel = require('../helpers/esbuildbabel')
+const babelPresetTypescript = require('@babel/preset-typescript')
```

Comment:
`@babel/preset-typescript` needs to be added as a dependency of package.json

Reply:
True, but the package was included on the dev branch already (since we have the same implementation for rollup already): https://github.com/rdkcentral/Lightning-CLI/blob/fix/typescriptEsBuild/package.json#L31